### DISCUSSION TITLE
Curso FullStack - M14A24 -Testando a Autenticacao

### DISCUSSION THREAD

**Discussion by Alekseidasilva** [balta-io/forum#587] [Tira Dúvidas] [answered]
Ola Balta, beleza?
Balta, estou executando os testes conforme a aula acima referida e quando acesso a pagina de logout, a tela fica piscado e o PC aumenta o consumo do VS2022!
Fiz um dubg no metodo OnInitializedAsync() da pagina de logout e notei que ele fica preso(loop) sobre este methodos e que faz com que o problema que reportei aconteca!
O que pode estar acontecendo, poderia me dar uma ajudinha?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Alekseidasilva, poderia por favor verificar se no código final do curso apresenta este problema? Assim já temos um direcionamento pra saber se é algo no código do curso ou no produzido durante as aulas.

Se for algo com seu código, pode postar uns trechos das telas aqui por favor?

**Reply by Alekseidasilva** [reply]
Boa tarde Balta! 
Olha incrível que pareça,  mas no código Master(do seu repo) não contém este erro. Farei questão de postar as telas e os trocos de códigos da mesma!

**Reply by Alekseidasilva** [reply]
@andrebaltieri aqui esta uma captura de tela e o codigo font da pagina de logout
https://github.com/balta-io/forum/assets/44441672/eee4fc33-b8d3-44f7-a460-03aa014fb886

**Reply by Alekseidasilva** [reply]
![image](https://github.com/balta-io/forum/assets/44441672/2d94245b-560d-47a8-a6cd-8fd4e76560bc)
![image](https://github.com/balta-io/forum/assets/44441672/07b9b8df-aefc-43a7-9bcb-cbb238013ce0)

**Reply by andrebaltieri** [reply]
@Alekseidasilva sem problemas... é só pra eliminarmos possíveis bugs... faz o seguinte...
1 - Limpa os dados do browser
2 - Fecha o visual studio
3 - Abre um terminal, navega para pasta da aplicação e executa os comandos:
```
dotnet clean
dotnet build
```

Roda tudo novamente... as vezes é algum lixo que ficou. Em adicional tenta depurar a aplicação para ver se encontra algum loop ou erro.

**Reply by laurolapajunior** [reply]
estou com esse mesmo problema, conseguiu resolver?

**Reply by laurolapajunior** [reply]
parece que chama a rota de logout, mas não está mantando o cookie apesar de ter retornado 204, ai ficar no loop,
